src/analyses/loop_termination.ml

```diff
@@ -0,0 +1,116 @@
+(** Work in progress *)
+
+open Analyses
+open GoblintCil
+open TerminationPreprocessing
+
+exception PreProcessing of string
+
+(*
+let loop_heads () =
+  let module FileCfg =
+  struct
+    let file = !Cilfacade.current_file
+    module Cfg = (val !MyCFG.current_cfg)
+  end in
+  let module WitnessInvariant = WitnessUtil.Invariant (FileCfg) in
+  WitnessInvariant.loop_heads (* TODO: Unused *)
+*)
```

Comment:
What is this commented out code good for?

Reply:
I considered using it later but probably am not going to do that. It will of course be removed in the end if it stays unused.